docs/prem-gateway/domains.md

```diff
@@ -0,0 +1,16 @@
+---
+id: domains
+title: Domains
+sidebar_label: Domains
+sidebar_position: 1
+---
+
+At start, the Prem App is accessible via IP that is printed by the installation script. This IP is Prem Gateway IP address.
+
+## How it works?
+
+Domains feature gives you option to easily map your domain to your Prem App application.
+It is necessary to insert A record for your subdomain(with wildcard) and point it to Prem Gateway IP address.
```

Comment:
we should mention here that actually two A records are needed, one for domain (example.com) and one for subdomain with wildcard (*example.com)